connect/src/spirc.rs

```diff
@@ -800,6 +889,39 @@ impl SpircTask {
                     let now = self.now_ms();
                     self.device.set_is_active(true);
                     self.device.set_became_active_at(now);
+                    self.player.emit_session_connect_event(
+                        self.session.connection_id(),
+                        self.session.username(),
+                        true,
+                    );
+                    self.player.emit_session_client_changed_event(
+                        self.session.client_id(),
+                        self.session.client_name(),
+                        self.session.client_brand_name(),
+                        self.session.client_model_name(),
+                    );
+
+                    let auto_play = matches!(
+                        &*self
+                            .session
+                            .get_user_attribute("autoplay")
+                            .unwrap_or_default(),
+                        "1"
+                    );
+
+                    self.player.emit_auto_play_changed_event(auto_play);
```

Comment:
Not sure I understand completely. If we transition from inactive to active state, does that always mean that autoplay, shuffle and repeat changed? Could it be that we were active once, then turned inactive, to now again become active with the same properties as before?

Reply:
I don't think so? But from a event consumer's perspective we basically don't exist if we're not active. And the switch from inactive to active could mean a different user so we should send those events.

Reply:
The idea is that an event consumer really doesn't need to maintain any sort of state if all it's doing is displaying info. And I think that the vast amount of event consumers will be doing just that. Just acting as dumb displays.

Reply:
Yeah for that usual case I think it should be fine. When we implement logout I do think this use case will be possible theoretically.

Reply:
The way the events are set up in this PR you could for sure use them to drive a dumb display. You wouldn't even need an event loop in the display code. The Eventhandler becomes the event loop, and there is no need for any shared state in the display either. You could even use the connection events to turn the display on and off.